docs/sphinx/user_guide/feature/view.rst

```diff
@@ -92,21 +92,56 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+Reshape
+^^^^^^^
+The ``RAJA::Reshape`` method will provide developers a ``RAJA::View`` given a pointer, dimentions and a meta-layout.
+Meta-layouts are predefined types which specify stride ordering. The first meta-layout is
+based on a right-most layout which follows standard C indexing (right most index has unit stride).
+The example below illustrates the following capability.
+
+.. literalinclude:: ../../../../examples/reshape.cpp
+   :start-after: _Reshape_right_start
+   :end-before: _Reshape_right_end
+   :language: C++
+In the example above a ``RAJA::View`` is returned with right most indexing having unit stride.
+
+The second supported meta-layout is the ``RAJA::layout_left`` which will follow a Fortran style indexing
+in which the left most index has unit stride and the right most has the longest stride. The configuration
+is demonstrated in the following example.
+
+.. literalinclude:: ../../../../examples/reshape.cpp
+   :start-after: _Reshape_left_start
+   :end-before: _Reshape_left_end
+   :language: C++
+
+Finally, users may specify a custom index ordering by providing a ``std::index_sequence`` list as a
+template argument.
+
+.. literalinclude:: ../../../../examples/reshape.cpp
+   :start-after: _Reshape_custom_start
+   :end-before: _Reshape_custom_end
+   :language: C++
+
+The example above will give unit stride to the right most index of the sequence while the left most index will have

```

Comment:
The example code for this has entry '0' for the second entry in the sequence. I think that requires a sentence of explanation.